scp.c

```diff
@@ -1262,8 +1262,14 @@ source(int argc, char **argv)
 	off_t i, statbytes;
 	size_t amt, nr;
 	int fd = -1, haderr, indx;
-	char *last, *name, buf[PATH_MAX + 128], encname[PATH_MAX];
+#ifdef WINDOWS
+	char* last, * name, * buf, * encname;
+	size_t encname_len, buf_len, tmp_len;
+#else
+	char* last, * name, buf[PATH_MAX + 128], encname[PATH_MAX];
```

Comment:
Please correct the spacing to improve the readability.

char* last, * name, buf[PATH_MAX + 128], encname[PATH_MAX]; -> char *last, *name, buf[PATH_MAX + 128], encname[PATH_MAX];

Reply:
same at line 1266, 1409, 1411